bigflow/build/spec.py

```diff
@@ -0,0 +1,262 @@
+"""Read and parse bigflow project configuration (setup.py / pyproject.toml)"""
+
+import functools
+import textwrap
+import typing
+import logging
+import typing
+import dataclasses
+import toml
+
+from pathlib import Path
+
+import setuptools
+
+import bigflow.resources
+import bigflow.version
+
+import bigflow.build.pip
+import bigflow.build.dev
+import bigflow.build.dataflow.dependency_checker
+
+import bigflow.commons as bf_commons
+
+
+logger = logging.getLogger(__name__)
+
+
+_ListStr = typing.List[str]
+
+# copied from `distutils.dist.DistributionMetadata`
```

Comment:
So why we wont use `from distutils.dist import DistributionMetadata`?

Reply:
Because such variable (list of strings) is not public (prefixed with "_").